src/test/java/io/swagger/swaggerhub/plugin/SwaggerHubUploadTest.java

```diff
@@ -471,6 +530,17 @@ private RequestPatternBuilder createPutSCMConfigRequestPatternWithAccountPATandP
 
     }
 
+    private RequestPatternBuilder createPutSCMConfigRequestPatternWithUrlAndProjectCollection(String owner, String api,String version, String oasVersion, String branch,
```

Comment:
Really small point, might be overkill - instead within this method you could `return createPutSCMConfigRequestPatternWithAccountPATandProject` and chain `.withRequestBody(matchingJsonPath("$.projectCollection"...` on there also. Would shorten the method a little.

Again it's no biggie.

Reply:
@michaelmelodysmartbear -- 
Would be better to do that way you are mentioned above, but Azure DevOps Server doesn't use `$.account` property which is configured in `createPutSCMConfigRequestPatternWithAccountPATandProject` method. In this case we'll need to add
```
<scmOrganization>account</scmOrganization>
```
property to the plugin configuration that is not what we need for TFS configs.